src/Services/GithubApiService.ts

```diff
@@ -120,33 +99,26 @@ export class GithubApiService extends GithubRepository {
         CONSTANTS.DEFAULT_GITHUB_RETRY_DELAY,
       );
       const response = await retry.fetch<Promise<T>>(async ({ attempt }) => {
-        const res = await soxa.post("", {}, {
-          data: { query: query, variables },
-          headers: {
-            Authorization: `bearer ${TOKENS[attempt]}`,
-          },
-        });
-        if (res?.data?.errors) {
-          return this.handleError(res?.data?.errors);
-        }
-        return res;
-      }) as QueryDefaultResponse<{ user: T }>;
+        return await requestGithubData(
+          query,
+          variables,
+          TOKENS[attempt],
+        );
+      });
 
-      return response?.data?.data?.user ??
-        new ServiceError("not found", EServiceKindError.NOT_FOUND);
+      return response;
     } catch (error) {
```

Comment:
The only time to enter this block is when an error occurs on the following lines.
https://github.com/ryo-ma/github-profile-trophy/blob/f887b9d62fbf047313ca415a090a1cb31529e864/src/Helpers/Retry.ts#L50

This error is not a Service Error, so even if the retry completes beyond the rate limit, the following error is returned.
line122 `return new ServiceError("not found", EServiceKindError.NOT_FOUND)`

Reply:
I haven't been able to Test it, so please let me know if I'm wrong.

Reply:
1 - Retry run
1.1 - Retry fail -> response will be null
1.2a - Retry fail again -> response will be ServiceError in Retry (GO TO 1.3)
1.2b - Retry success -> response will be the data (responseData.data.user)

1.3 -> retry fails twice -> response will be GithubErrorResponse | GithubExceedError

Reply:
On which line does 1.2a give an error?

Reply:
1. Retry -> line 22

Reply:
It is still on the Retry, and I think it goes to the Retry catch.

Reply:
I'll try testing it tomorrow.